docs/usage/java.md

```diff
@@ -131,3 +131,68 @@ module.exports = {
   ],
 };
 ```
+
+### Google Artifact Registry
+
+There are multiple ways to configure Renovate to access Artifact Registry.
+
+#### Using Application Default Credentials / Workload Identity (Self-Hosted only)
+
+Just configure [ADC](https://cloud.google.com/docs/authentication/provide-credentials-adc) /
+[Workload Identity](https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity) as normal and _don't_
+provide a username, password or token. Renovate will automatically retrieve the credentials using the
+google-auth-library.
+
+##### Using long-lived service account credentials
```

Comment:
Presumably, it's not intended that this heading is a child of `Using Application Default Credentials / Workload Identity (Self-Hosted only)`

Reply:
No, sorry, completely missed that! Thanks for spotting it!